src/plugins/cordova-plugin-vibration/app-host.js

```diff
@@ -87,8 +87,9 @@ module.exports = function (messages) {
 
         for (x = 1; x <= times; x++) {
             body.animate({ left: -10 }, 5,
-                function () { checkForVibration(deferred, body);
-            })
+                function () {
```

Comment:
So these two variants make the linter happy:
1) invocations indented consistently
```javascript
body
    .animate({ left: -10 }, 5, function () {
        checkForVibration(deferred, body);
    })
    .animate(...)
```
or one-liner function:
2)
```javascript
body.animate(..., function () { checkForVibration(deferred, body); })
    .animate(...)
```
In this case I opted for the latter one (since the function body is short).